testutil/server.go

```diff
@@ -107,6 +107,8 @@ type MockMetricsServer struct {
 	StreamData map[string][]*gopb.SystemMetricsData
 	Close      *gopb.SystemMetricsSeriesEnd
 	DialOpts   timber.DialCedarOptions
+
+	*gopb.UnimplementedCedarSystemMetricsServer
```

Comment:
I don't really think it matters, but should this be a pointer or not? It seems like you used a non-pointer field for the other unimplemented servers (in Cedar/Jasper/Poplar).

Can you add the same comment about forward compatibility?

Reply:
Yeah it doesn't matter, idk why I made them pointers. Done.